app/assets/stylesheets/_home.scss

```diff
@@ -137,5 +137,6 @@ $bgFeatureBlock: #fffbf2 !default;
 .btn-search-inverse-cities {
   line-height: 100%;
   margin: 0;
+  padding: 13px 14px;
   width: 100%;
 }
```

Comment:
Actually this makes the switch button too big. It's not the main attraction here.
![oversize](https://user-images.githubusercontent.com/540760/29436440-d0439de0-83ab-11e7-9ce5-963dc3cc0a13.png)

I like the short : 
.btn-search-inverse-cities {
   line-height: 100%;
 }